### ISSUE TITLE
`bottom` and `right` show wrong UI lines in Gcss simulator (but `left` and `top` correct)

### ISSUE THREAD

**Issue by Laaouatni** [Laaouatni/gcode.js#35]
**Describe the bug**
`right`, and `bottom` in MoveTo don't work in the CSS generator

they need to be attached, and the number ::before need to be in the correct x,y (displayed in the popup)

**Screenshots**

> ❌ 
```javascript
for (let i = 0; i < 9; i++) {
  a.moveTo({
    left: 50,
    bottom: i * 10,
  });
}
```

![image](https://user-images.githubusercontent.com/87947051/186006731-c9b3b4ba-e0e3-4ce7-9e3f-156d536ae467.png)

-----

but with `top` is ok ✅
![image](https://user-images.githubusercontent.com/87947051/186007161-dedf6a0d-d80e-4e18-857a-f4b7c89a0750.png)

----

 > ❌ there is a bug with `right`

```javascript
for (let i = 0; i < 9; i++) {
  a.moveTo({
    right: 50,
  });
}
```

![image](https://user-images.githubusercontent.com/87947051/186007345-a886a452-b7db-479d-8a1d-8fe7b40041a7.png)

-----

> ❌ also here the same bug with `right`

```javascript
for (let i = 0; i < 9; i++) {
  a.moveTo({
    right: 50,
    top: i * 10,
  });
}
```

![image](https://user-images.githubusercontent.com/87947051/186007371-f264e19d-f711-4678-9a2d-5fc143d08a06.png)

**Comment by Laaouatni** [comment]
there is a bug in animation we need to solve, the positioning is correct

**Comment by Laaouatni** [comment]
solved in #37 

![](https://user-images.githubusercontent.com/87947051/186211372-49ff908a-9f42-41f8-8d20-94e325690b2a.gif)

**Comment by Laaouatni** [comment]
🐛another time, I find another bug! 

if you use `top`, with `right` and the values aren't 45degres, this happen
![image](https://user-images.githubusercontent.com/87947051/186213658-ff865fad-a7f3-4ddf-88e2-039f030b309f.png)

the same for `bottom` and `left`
![image](https://user-images.githubusercontent.com/87947051/186214060-adadfef2-a8db-4128-8c9b-8e7bafaa7470.png)

**Comment by Laaouatni** [comment]
the left and bottom can be solved by getting the negative value,
`-1 * this.obj.lineAngle`

but for now I can't figure out the formula for the `right`/`top`

here the table of my checkings but I think there is some calculation that it will make us get the third column value

| right | angle | the value that correct the angle UI |
|-----|----|-----|
| 5   | 84 | 96  |
| 10  | 78 | 102 |
| 20  | 68 | 112 |
| 30  | 59 | 122 |
| 40  | 51 | 129 |
| 50  | 45 | 135 |
| 60  | 39 | 141 |
| 70  | 35 | 145 |
| 80  | 32 | 148 |
| 90  | 29 | 151 |
| 100 | 26 | 154 |
| 110 | 24 | 156 |
| 120 | 22 | 158 |
| 130 | 21 | 159 |
| 140 | 19 | 161 |
| 150 | 18 | 162 |

**Comment by Laaouatni** [comment]
I figure it out now! all the angles, if you sum them if it will give you 180.

now if we do `180 - this.obj.lineAngle` we will solve that

solved in #38
